packages/frontend/src/pages/home/Home.tsx

```diff
@@ -96,12 +98,48 @@ export function Home(props: HomeProps) {
               transaction.status,
               formatHashLong(transaction.hash),
               formatCurrencyUnits(transaction.amount, transaction.assetId),
-              <AssetNameCell assetId={transaction.assetId} />,
+              <AssetCell assetId={transaction.assetId} />,
               transaction.positionId.toString(),
             ],
           }
         })}
       />
+      <div className="mb-1.5">
+        <span className="float-left font-medium text-lg">
+          Latest forced trade offers
+        </span>
+        <SimpleLink className="float-right" href="/forced/offers">
+          view all
+        </SimpleLink>
+      </div>
+      <Table
+        noRowsText="there is no active offers at the moment"
+        columns={[
+          { header: 'Type' },
+          {
+            header: 'Asset',
+            numeric: true,
+            textAlignClass: 'text-left',
+            fullWidth: true,
+          },
+          { header: 'Price', numeric: true },
+          { header: 'Total', numeric: true },
+          { header: 'Position ID', numeric: true },
+        ]}
+        rows={props.forcedTradeOffers.map((offer) => {
+          const link = `/forced/offers/${offer.id}`
+          return {
+            link,
+            cells: [
+              offer.type,
+              <AssetCell assetId={offer.assetId} amount={offer.amount} />,
+              formatCurrencyUnits(offer.price, 'USD'),
+              formatCurrencyUnits(offer.total, AssetId.USDC),
```

Comment:
I think that the designs imply displaying the price in USD in both cases. But we are storing the amounts in USDC, so it should probably look sth like this 
```suggestion
              formatCurrency(offer.price / 1000n, 'USD'),
              formatCurrency(offer.total / 1000n, 'USD'),
```
Price is also formatted like this, so there should not be a problem with precision.

Reply:
It doesn't look right rn, the numbers don't really add up
<img width="999" alt="Screenshot 2022-05-19 at 11 17 32" src="https://user-images.githubusercontent.com/38423054/169259168-bb89727f-aa3f-4593-bc0c-93b8c74b7ff0.png">

Reply:
@sz-piotr mentioned that price should be displayed in USD and total in USDC - designs are not correct anymore if that decision stays.